plugins/pmsmart/build.gradle

```diff
@@ -8,6 +8,8 @@ dependencies {
     implementation "io.github.bonigarcia:webdrivermanager:4.4.3"
     
     testImplementation project(':logic')
+    testImplementation project(':logic').sourceSets.test.output
```

Comment:
This looks like a hack. If necessary, please move classes used by tests to a common test util project.

Reply:
done - seems to have become obsolete in the meantime anyway 😃